docs/src/model_docs/vertical/sbm.md

```diff
@@ -425,8 +425,8 @@ the figure below where the blue line represents the ``K_{sat}`` value.
 
 ### Infiltration
 
-The water available for infiltration is taken as the rainfall including meltwater.
-Infiltration is determined separately for the compacted and non compacted areas, as these
+The water available for infiltration is taken as the rainfall including melted water.
```

Comment:
My preference is to use meltwater here (it is a common used term).

Reply:
Done